### PR TITLE
Make %mentions actually look like mentions

### PR DESCRIPTION
**Pull request by sipec**
Screen Shot 2022-10-03 at 5 28 14 PM
Screen Shot 2022-10-03 at 5 30 13 PM

### CODE REVIEW

**Comment by sipec** [comment]
I know this is gonna be controversial. but the previous card mention experience wasn't actually that great when doing anything other than making a list of cards. I do think we should re-add it later..

File: web/components/contract/contract-mention.tsx
Comment (akrolsmir) [inline web/components/contract/contract-mention.tsx:19]: Can we use the tippy Tooltip instead of just title? For consistent formatting with all the rest of our tooltips

File: web/components/contract/contract-mention.tsx
Comment (akrolsmir) [inline web/components/contract/contract-mention.tsx:28]: I like the look of this, though would add a bit more padding in the x direction to make it look more pill-like. px-2 or px-2.5 or px-3? Idk, play with it a bit

**Review by akrolsmir — APPROVED** [review]
Looks much neater inline, and love the little % pill at the end. Great idea!

I'm not sure I like how this breaks existing cards, but also probably not that many people have used contract cards/% mentions so far? Probably just give a heads-up in standup before merging this in, I think @SirSaltyy and @FRCassarino have used contract cards before.

Longer term, I'm not sure if I like the [Notion pattern](https://manifoldmarkets.notion.site/Notion-al-markets-master-plan-b6863be92e784fa6a28f18bea5ff3835) of having a second popup to choose whether you want to embed as a mention or link, because it adds more friction to selecting stuff. But I don't know of great UX alternatives either. Don't feel too strongly, I'll defer to your thoughts here

File: web/components/contract/contract-mention.tsx
Comment (akrolsmir) [reply web/components/contract/contract-mention.tsx:19]: Saw your thoughts `[c] tooltips were too distracting, just made it a title attribute (yes I know this sucks)`

Which is fine, I guess. Actually the best thing would be to show the contract card in a popup on hover, gwern-style (which shouldn't be too hard)

**Pull request merged**
